what-job-react/src/components/NewPostPage.js

```diff
@@ -6,31 +6,37 @@ export function NewPostPage() {
     const [intro, setIntro] = useState("Short introduction for yourself, how long you have worked in this position and an example of a great day and a bad day at your job.");
     const [day_to_day, setDay_to_day] = useState("Go over the boring/exciting stuff that you often overlook but most of your days consist of");
     const [career_path, setCareer_path] = useState("How did you get to this position, study, luck, hardship, Nepotism. As well as ways that you know that enable a person to enter this field today.");
+    const [industry_growth, setIndustry_growth] = useState("How fast is this industry growing, or is it shrinking? do they have good or bad entry into the industry due to new things happening?.");
```

Comment:
Maybe put these in as placeholer text? https://www.w3schools.com/tags/att_input_placeholder.asp

Then you can default to null